ts/boot/scheduleLocalNotifications.ts

```diff
@@ -44,7 +43,24 @@ export const scheduleLocalNotificationsAccessSpid = () => {
  * Remove all the local notifications relating to authentication with spid
  */
 export const removeScheduledNotificationAccessSpid = () => {
-  PushNotification.cancelLocalNotifications({
-    id: FIRST_ACCESS_SPID_TAG
-  });
+  /**
+   * With the current library version seems that cancelLocalNotifications doesn't work.
```

Comment:
```suggestion
   * With the current library version (7.2.0) seems that cancelLocalNotifications doesn't work.
```
Could it be useful when we'll investigate about this problem?

Reply:
Sure, updated in 28775d7145c985e6d52f51289f91abe6bb2a1ef1